enterprise/backend/src/metabase_enterprise/tenants/api.clj

```diff
@@ -50,8 +51,9 @@
   [{id :id} :- [:map {:closed true} [:id ms/PositiveInt]]
    _query-params
    tenant :- [:map {:closed true}
-              [:name {:optional true} [:maybe ms/NonBlankString]]
```

Comment:
github won't let me comment on it, but the docstring above is no longer accurate